express-app/controllers/user.js

```diff
@@ -59,6 +66,7 @@ module.exports = {
    * @param {object} response - response object.
    */
   requestUser: function (data, request, response) {
+    var self = this;
```

Comment:
Because the configuration change and if a used this the application send an error. Thats why use self insted this.